prog-o-meter.py

```diff
@@ -143,23 +150,30 @@ def add_day(self):
         self.days_remaining = self.GOAL - self.days
         self.completion_date = self.get_completion_date(self.days_remaining)
         self.canvas.itemconfig(self.rectangle_list[self.days-1], fill = "green")
+        self.canvas.itemconfig(self.encourage_text, text = "".join(self.encourage[self.new_no(self.x)]))

```

Comment:
I think instead of already adding the encouragement when the user opens the program, this is the location where encouragements should pop up, so instead of item config, this is where we should actually introduce the line of code, which is currently in line 83. 
Another suggestion I have would be to add this line of code at another location on the canvas, since there is so much text up top that this doesn't get a lot of focus. It is a really nice feature, so I think it should get more focus. Maybe we should have this text show up below the prog-o-meter timeline? What do you think about that?

Reply:
Do you want me to add repeat the encouragement below the timeline or want it to be placed only below the timeline?

Reply:
Also is it fine if I add it inside the existing prog_o_meter() function?

Reply:
I think we should have it placed only below the timeline. I also think you should not add it the the prog-o-meter function, but just have it here in the add_day function, since the text should pop up when the add_day button is clicked. So basically what I would do, is replace this line with line 83 (and change the x,y coordinated given in line 83, so that the text shows up below the timeline.)
Let me know if you have any more questions about it :)

Reply:
I won't be able to add a canvas.createtext() in the add_day since it would create a new text everytime the button is clicked. Instead I assigned a new variable below VERTICAL_TEXT_POSITION. Is it fine?